storybook/storybook-react/src/Grid/Grid.stories.tsx

```diff
@@ -31,66 +68,96 @@ const StoryTemplate: Story = {
   ],
 }
 
-export const Default: Story = {
-  ...StoryTemplate,
+const GridCellStoryTemplate: GridCellStory = {
+  argTypes: gridCellArgTypes,
+  decorators: [
+    (Story) => (
+      <Screen>
+        <Grid>
+          <Story />
+        </Grid>
+      </Screen>
+    ),
+  ],
+  render: ({ children, ...args }) => <Grid.Cell {...args}>{children}</Grid.Cell>,
+}
+
+export const Default: GridStory = {
```

Comment:
As a developer, the first thing I'd want to see in the docs is the grid with one cell spanning the entire grid, together with the controls for the grid and that one cell. That would get you directly to the code you need. The code in this example will usually not be useful for a dev.

Reply:
The current examples form a walkthrough of the features of grid and grid cell. Yes, that’s a different approach than what you describe.

Not sure how an example with one cell would get you directly to ‘the code you need’. Can you write the story as you see it?

Reply:
Discussed this in person, we'll leave it as is for now.